pickle.gemspec

```diff
@@ -26,7 +26,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency "bundler"
   s.add_development_dependency "git"
   s.add_development_dependency "yard"
-  s.add_development_dependency "rspec-rails", "~>3.0"
+  s.add_development_dependency "rspec-rails", ">= 3.0"
+  s.add_development_dependency "rspec-mocks", ">= 3.12.4"
```

Comment:
Required for rails 7.1 for this fix: https://github.com/rspec/rspec-mocks/commit/a2af97db4799477463fcc75d73373eb64f63ba83